### ISSUE TITLE
Entity is using state class 'measurement' which is impossible considering device class ('water') it is using

### ISSUE THREAD

**Issue by R1DEN** [barleybobs/homeassistant-ecowater-softener#47]
I have the following warning displayed in my log, so will report it here
```
Entity sensor.ecowater_acxxxxx_water_available (<class 'custom_components.ecowater_softener.sensor.EcowaterSensor'>) is using state class 'measurement' which is impossible considering device class ('water') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/barleybobs/homeassistant-ecowater-softener/issues
```

**Comment by BillyFKidney** [comment]
I was receiving the same error:
```
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:528
Integration: Sensor (documentation, issues)
First occurred: 7:44:44 PM (1 occurrences)
Last logged: 7:44:44 PM

Entity sensor.ecowater_err_3700_water_available (<class 'custom_components.ecowater_softener.sensor.EcowaterSensor'>) is using state class 'measurement' which is impossible considering device class ('water') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/barleybobs/homeassistant-ecowater-softener/issues
```

I changed FILE.NAME to contain the following:
```
SENSOR_TYPES: tuple[EcowaterSensorEntityDescription, ...] = (
    EcowaterSensorEntityDescription(
        key=STATUS,
        name="Status",
        icon="mdi:power",
    ),
    EcowaterSensorEntityDescription(
        key=WATER_AVAILABLE,
        name="Water Available",
        icon="mdi:water",
        device_class=SensorDeviceClass.WATER,
        state_class=SensorStateClass.TOTAL,
```

**Comment by barleybobs** [comment]
Fixed in v3.1.0
